### PR TITLE
Unable connect to db #46

### PR DESCRIPTION
**Pull request by Oksana1788**

### CODE REVIEW

File: src/main/resources/application.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+    driver-class-name: com.mysql.jdbc.Driver
+    url: jdbc:mysql://localhost:3306/ipcount-core
+    username: root
+    password:
+  data:
+    jpa:
+      repositories:
+        enabled: true
+  jpa:
+   database-platform: org.hibernate.dialect.MySQL5InnoDBDialect
```
Comment (yurii-polishchuk) [inline src/main/resources/application.yml]: let me clarify if on production we are using InnoDB for MySQL.

File: src/main/resources/application.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  datasource:
+    driver-class-name: com.mysql.jdbc.Driver
+    url: jdbc:mysql://localhost:3306/ipcount-core
+    username: root
+    password:
+  data:
+    jpa:
+      repositories:
+        enabled: true
+  jpa:
```
Comment (yurii-polishchuk) [inline src/main/resources/application.yml]: Please take a look at those properties (read about them and add to config)

```
  jpa:
    show-sql: false
    generate-ddl: true
    hibernate:
      ddl-auto: validate
    properties:
      hibernate:
        dialect: ??
```

File: src/main/resources/application.yml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+spring:
+  datasource:
```
Comment (yurii-polishchuk) [inline src/main/resources/application.yml]: in data source also please take a look at those parameters (read about them):

```
    max-wait: 5000
    max-active: 200
    min-idle: 25
    test-on-borrow: true
    validationQuery: SELECT 1
```

File: src/main/resources/application.yml (hunk 1 above)
Comment (Oksana1788) [inline src/main/resources/application.yml]: @yurii-polishchuk If you have no objection , then yes

Comment (yurii-polishchuk) [inline src/main/resources/application.yml]: @Oksana1788 I meant that I would need to check real DB that is already running :) I will follow up on this today.

Comment (Oksana1788) [inline src/main/resources/application.yml]: @yurii-polishchuk Sorry I  misunderstood

**Pull request merged**